App.tsx

```diff
@@ -1,21 +1,21 @@
-import "expo-dev-client";
-import "./polyfills";
-import {
-  configureReanimatedLogger,
-  ReanimatedLogLevel,
-} from "react-native-reanimated";
 import { configure as configureCoinbase } from "@coinbase/wallet-mobile-sdk";
 import DebugButton from "@components/DebugButton";
 import { BottomSheetModalProvider } from "@design-system/BottomSheet/BottomSheetModalProvider";
+import { useReactQueryDevTools } from "@dev-plugins/react-query";
```

Comment:
What import cleanup rule do you have? I've used prettier import cleanup before - maybe we should automate and consolidate our different import sorting strategies

Reply:
<img width="466" alt="Screenshot 2025-01-21 at 8 27 46 AM" src="https://github.com/user-attachments/assets/3e96da6e-c48b-4505-83b5-f49843f0a632" />